packages/validation-manager/src/decodeHelper.ts

```diff
@@ -0,0 +1,299 @@
+import { ERC7562Call } from './ERC7562Call'
+import Debug from 'debug'
+import { IEntryPoint__factory, IPaymaster__factory, SenderCreator__factory } from '@account-abstraction/utils'
+import { FunctionFragment, Interface } from 'ethers/lib/utils'
+import { IRip7560Account__factory, IRip7560Paymaster__factory } from '@account-abstraction/utils/dist/src/types'
+
+const debug = Debug('aa.dump')
+
+export function base64Tohex (input: string): string {
```

Comment:
Are we the ones who encoded the data in `base64` in native 7562 tracer in the first place, or does it come from `go-ethereum` developers?